content/riak/kv/2.2.0/developing/usage/search.md

```diff
@@ -220,6 +223,126 @@ only *one* Riak bucket, you can set the `search_index` property on that
 bucket. If more than one bucket is to share a Solr index, a bucket type
 should be used. More on bucket types in the section directly below.
 
+### Associating via Bucket Type
+
+In Riak versions 2.0 and later, Basho suggests that you use [bucket
```

Comment:
Change:
"In Riak versions 2.0 and later, Basho suggests that you" to
"We suggest that you"